playwright/tenor/README.md

```
@@ -0,0 +1,117 @@
+// Til (midlertidig, uten Tenor):
+delegator = { pid: '03906197811', navn: 'STRAFFET KOST', etternavn: 'KOST' };
+recipient = { pid: '23854897845', navn: 'KONSERVATIV FATTIGMANNSKOST', etternavn: 'FATTIGMANNSKOST' };
+```
+
+Verdiene under er de opprinnelige hardkodede testdataene som ble erstattet – de
+var reelle, fungerende aktører i TT02 og er derfor gode utgangspunkt. (De kan ha
+blitt endret/slettet siden; verifiser i [Tenor](https://www.skatteetaten.no/skjema/testdata/)
+eller Altinns testdata-verktøy.)
+
+### Bosatte, myndige personer (`bosattMyndigPerson` / `bosatteMyndigePersoner`)
+| pid | navn |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)